bin/generate_unfiltered_sce.R

```diff
@@ -122,8 +140,12 @@ if (opt$feature_dir != "") {
     include_unspliced = FALSE,
     fry_mode = TRUE,
     feature_data = TRUE,
+    tech_version = opt$technology,
+    assay_ontology_term_id = opt$assay_ontology_term_id,
+    seq_unit = opt$seq_unit,
     library_id = opt$library_id,
-    sample_id = sample_ids
+    sample_id = sample_ids,
+    project_id = project_id
```

Comment:
I don't think this exists? Should it be..

```suggestion
    project_id = opt$project_id
```